### ISSUE TITLE
Worktable - accept items from hoppers

### ISSUE THREAD

**Issue by erima** [ForestryMC/ForestryMC#489]
I was just thinking that worktables should accept items from hoppers and pipes as it would be useful for recipes that require smelted items or items from Buildcraft Auto Workbenches. Conceptually it makes sense as there's a chest in the recipe for the Worktable.

**Comment by CovertJaguar** [comment]
Not saying I necessarily agree, but I think that was intentional on @SirSengir 's part.

**Comment by SirSengir** [comment]
I honestly don't remember whether that was a conscious decision. Not sure whether hoppers existed at that point. Or maybe I didn't think it was necessary since the table has a large internal inventory and can't be automated.

There may be an issue with multiple hoppers feeding into the table and possibly congesting the internal inventory with a single ingredient. (Though the same can happen when stuff is piped in.)

tl;dr: If it was intentional I don't remember the reasoning.

**Comment by CovertJaguar** [comment]
So...fair game!

**Comment by mezz** [comment]
It seems like it would be really tough to hopper things in without clogging it with something.
I would restrict it so that it only accepts items that are already in there, but it would mess up when that item runs out.
Thoughts?

**Comment by SandGrainOne** [comment]
That is something the logistics systems can solve. Even BuildCraft can feed items into an inventory at a spesific ratio.

**Comment by mezz** [comment]
Sounds good! Added in the latest unstable.
